pangea-sdk/service/ip_intel/api.go

```diff
@@ -40,6 +40,40 @@ func (e *IpIntel) Lookup(ctx context.Context, input *IpLookupRequest) (*pangea.P
 	return &panresp, err
 }
 
+// IP address geolocate
+//
+// Geolocate an IP address.
+//
+// Example:
+//
+//	 input := &ip_intel.IpGeolocateRequest{
+//	     Ip: "93.231.182.110",
+//	     Raw: true,
+//	     Verbose: true,
+//	     Provider: "crowdstrike",
+//	 }
+//
+//		checkOutput, _, err := ipintel.geolocate(ctx, input)
```

Comment:
I think we can remove the extra tab here? Unless this was something `gofmt` did?
```suggestion
//	checkOutput, _, err := ipintel.geolocate(ctx, input)
```